packages/extension-api/src/extension-api.d.ts

```diff
@@ -17,6 +17,11 @@
  ***********************************************************************/
 
 declare module '@podman-desktop/api' {
+  /**
+   * The version of Podman Desktop.
+   */
+  export const version: string;
```

Comment:
I'm wondering if we respect SemVer if we should not expose the SemVer components so that extensions have a better way to implement checks ?

Reply:
it can be any string (as it relies on the definition of package.json version definition)

As some people may override, build a custom version, it may not be a semVer at all

so we should not probably restrict here